backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/DeviceManagementTest.java

```diff
@@ -134,4 +144,39 @@ private Map<String, Object> getUpdateDeviceTypeInput(UUID internalId) {
 
     return Map.of("input", input);
   }
+
+  @Test
+  void markDeviceTypeAsDeleted_uuid_success() {
```

Comment:
Thank you for adding the mutation tests here! Definitely a good habit to get into